<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedPrinterWidget</class>
 <widget class="QWidget" name="SharedPrinterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>38</horstretch>
    <verstretch>44</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="buddy">
        <cstring>actionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <property name="currentText">
        <string>Create</string>
       </property>
       <item>
        <property name="text">
         <string>Create</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="deleteCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Delete all shared printer connections</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="sharedGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>38</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="title">
      <string>Shared printer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="sharedPathLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Shared path:</string>
        </property>
        <property name="buddy">
         <cstring>sharedPathLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="sharedPathLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="sharedPathToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="setPrinterCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Set this printer as the default printer...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="onlyIfCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>... only if a local printer is not present</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>75</width>
          <height>49</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mapGroupBox">
     <property name="title">
      <string>Map to local port (optional)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="localPortLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Local port:</string>
        </property>
        <property name="buddy">
         <cstring>localPortComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="localPortComboBox">
        <item>
         <property name="text">
          <string>LPT1:</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LPT2:</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LPT3:</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LPT4:</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="unmapCheckBox">
        <property name="text">
         <string>Unmap all local ports</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="reconnectCheckBox">
        <property name="text">
         <string>Reconnect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>deleteCheckBox</tabstop>
  <tabstop>sharedPathLineEdit</tabstop>
  <tabstop>sharedPathToolButton</tabstop>
  <tabstop>setPrinterCheckBox</tabstop>
  <tabstop>onlyIfCheckBox</tabstop>
  <tabstop>localPortComboBox</tabstop>
  <tabstop>reconnectCheckBox</tabstop>
  <tabstop>unmapCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>setPrinterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>onlyIfCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
